misc/python/materialize/mzcompose/composition.py

```diff
@@ -360,8 +374,10 @@ def invoke(
                         bufsize=1,
                     )
             except subprocess.CalledProcessError as e:
-                if e.stdout:
+                if e.stdout and not capture_and_print:
```

Comment:
Can you explain why `not capture_and_print`?

Reply:
We already print out the output during the execution in lines 350+353. If we'd print it again, then it's duplicated.

Reply:
Makes sense, thanks